include/aspect/global.h

```diff
@@ -190,6 +190,21 @@ namespace aspect
   extern const bool output_parallel_statistics;
 
 
+  namespace citation
```

Comment:
Thinking about this, I think it would be nicer to make this a class `CitationInformation`, so that we don't introduce global variables. The only global variables we have in aspect are ones that are initialized once at the beginning and do not carry any dynamic state.